content/livesync/models.textile

```diff
@@ -141,17 +218,17 @@ The event passed to the merge function can be either confirmed or optimistic:
 
 The SDK might call the merge function multiple times with the same state and event; to rebase optimistic events onto the newly confirmed state. It's essential that the merge function, when executed with identical inputs, consistently produces the same output. The merge function should be pure and deterministic, not depending on any external state.
 
-h3(#buffer). Event buffer
+h3(#event-buffer). Event buffer
 
 Change events from your backend are delivered to the SDK as messages over the "channels":/channels. By default, messages are processed by the SDK in the order in which they are received.
 
-The SDK supports a short buffering time buffering for change events, facilitating short-term reordering and de-duplication. By default, his feature is not enabled. To activate it, set the @eventBufferOptions.bufferMS@ in the "@ClientOption@":/livesync/models#options to a non-zero value.
+The SDK supports a short buffering time buffering for change events, facilitating short-term reordering and de-duplication. By default, his feature is not enabled. To activate it, set the @eventBufferOptions.bufferMS@ in the "@ClientOption@":#client-options to a non-zero value.
 
 When using the event buffer, change events are de-duplicated and ordered according to their "@message.ID@":/api/realtime-sdk/messages#id, which corresponds to the change event's "@sequenceId@":#sequence.
 
-By default, the events in the buffer are ordered numerically if the @message.ID@ can be coerced to a number. Otherwise, events will be ordered lexicographically by their @message.ID@. You can specify a custom ordering based on any part of the message via the @eventBufferOptions.eventOrderer@ "@ClientOption@":/livesync/models#options.
+By default, the events in the buffer are ordered numerically if the @message.ID@ can be coerced to a number. Otherwise, events will be ordered lexicographically by their @message.ID@. You can specify a custom ordering based on any part of the message via the @eventBufferOptions.eventOrderer@ "@ClientOption@":#client-options.
```

Comment:
```suggestion
By default, the events in the buffer are ordered numerically if the @message.id@ can be coerced to a number. Otherwise, events will be ordered lexicographically by their @message.id@. You can specify a custom ordering based on any part of the message via the @eventBufferOptions.eventOrderer@ "client option":#client-options.
```